symfony/plugins/orangehrmAdminPlugin/Dao/LocationDao.php

```diff
@@ -38,7 +38,11 @@ class LocationDao extends BaseDao
     public function getLocationById(int $locationId): ?Location
     {
         try {
-            return $this->getRepository(Location::class)->find($locationId);
+            $location = $this->getRepository(Location::class)->find($locationId);
```

Comment:
What happen if locationid <= 0 ? throes DAO exceptions ?

Reply:
In such a case, the function will return null. No exceptions will br thrown.